Dockerfile

```diff
@@ -37,6 +37,20 @@ ADD clean-logs.sh /clean-logs.sh
 # Add startup script
 ADD virtuoso.sh /virtuoso.sh
 
+# Let's add some other dependencies
+RUN apt-get install -y python2.7 python-pip awscli
+
+ADD crontab /etc/cron.d/periodic_back_cron
+
+RUN chmod 0644 /etc/cron.d/periodic_back_cron
+
+#ADD periodic_backup.sh /periodic_backup.sh
+#RUN chmod u+x /periodic_backup.sh
+
+RUN touch /var/log/cron.d
```

Comment:
So my understanding of this is as follows - sometimes if the timestamp of the cron.d file matches the timestamp when the system was built, then it will not see the additional scripts as being added - and not load the cron jobs.  By touching the cron.d file, it makes sure that the timestamp is different.